### PR TITLE
[FIX] stock_picking_report_valued: Wrong taxes amount if round globally

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

File: stock_picking_report_valued/models/stock_picking.py [hunk 1]
```diff
@@ -41,12 +41,30 @@ def _compute_amount_all(self):
         records...).
         """
         for pick in self:
-            amount_untaxed = sum(pick.move_line_ids.mapped(
-                'sale_price_subtotal'))
-            amount_tax = sum(pick.move_line_ids.mapped(
-                'sale_price_tax'))
+            round_curr = self.sale_id.currency_id.round
+            amount_untaxed = amount_tax = 0.0
+            for tax_id, tax_group in self.get_taxes_values().items():
```
Comment (rousseldenis) [inline stock_picking_report_valued/models/stock_picking.py]: Maybe this computation on rounding should depends on config parameter (round per line or round globally) as the result should be different.

File: stock_picking_report_valued/models/stock_picking.py [hunk 2]
```diff
@@ -41,12 +41,30 @@ def _compute_amount_all(self):
         records...).
         """
         for pick in self:
-            amount_untaxed = sum(pick.move_line_ids.mapped(
-                'sale_price_subtotal'))
-            amount_tax = sum(pick.move_line_ids.mapped(
-                'sale_price_tax'))
+            round_curr = self.sale_id.currency_id.round
```
Comment (rousseldenis) [inline stock_picking_report_valued/models/stock_picking.py]:
```suggestion
            round_curr = pick.sale_id.currency_id.round
```
and so on.

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_picking_report_valued/models/stock_picking.py (hunk 1 above)
Comment (carlosdauden) [reply stock_picking_report_valued/models/stock_picking.py]: Changes done

File: stock_picking_report_valued/models/stock_picking.py (hunk 2 above)
Comment (carlosdauden) [reply stock_picking_report_valued/models/stock_picking.py]: Ups...
Changes done

Comment (rousseldenis) [reply stock_picking_report_valued/models/stock_picking.py]: > and so on.

Missed the next self

Comment (carlosdauden) [reply stock_picking_report_valued/models/stock_picking.py]: You are right, changed.
Thanks.

**Review by pedrobaeza — COMMENTED** [review]
Travis is red. Seeing this, why don't we ignore stock.move.line fields and directly compute things at stock.move level and only checking qty at sml level?

**Comment by pedrobaeza** [comment]
@carlosdauden what do you think about my thoughts?

**Comment by carlosdauden** [comment]
@pedrobaeza 
As what is printed on the delivery note are stock.move.line if we computed the fields in stock.move then we would have to recalculate the proportional part of each printed line.
Another thing is that the Odoo core report printed the stock.move and added the lot inside `<td>`, which is how I think it should be...

**Review by pedrobaeza — APPROVED** [review]
OK, got it

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**